### ISSUE TITLE
Eigenen WLAN Hotspot abschalten

### ISSUE THREAD

**Issue by copacetic82** [goecharger/go-eCharger-API-v2#35]
Es wäre nützlich den eigenen WLAN Hotspot per API oder App abschalten zu können, oder sobald der go-echarger sich mit dem eigenen WLAN verbindet. Aktuell sind die hotspots permanent aktiv, was bei mehreren go-echargern zu unnötigen Störungen im WLAN führt.

**Comment by syssi** [comment]
Sollte es keine Option sein den AP-Modus vollständig zu deaktivieren und (im Notfall) über den Button wieder zu triggern, wäre es bereits hilfreich, wenn man täglich eine Instruktion an die Wallbox senden könnte um den AP-Modus zu beenden / gestoppt zu halten. Bleibt diese Instruktion mal aus, so geht der AP-Modus wieder an.

**Comment by copacetic82** [comment]
Soetwas würde mir auch reichen. Ich habe nach wie vor meine 3 aktiven störenden hotspots, die hoffnung stirbt zuletzt.

**Comment by povlhp** [comment]
Another option, turn the built-in WLAN off if it is connected to another WLAN. Most other equipment I have do this.
But maybe allow to re-enable for x minutes with a long press of the button (5-10s ?)
And one more thing, please make sure the built-in WLAN is 20MHz bandwidth only. No need for it to pollute too many channels with 40MHz.

**Comment by syssi** [comment]
#### 17.03.22, 08:36

Sehr geehrte Frau M.,

ich waere ihnen sehr dankbar, wenn sie dieses Anliegen ("Accesspoint-Modus deaktivierbar machen") an das Entwicklerteam weitergeben. Ich bin mir sehr sicher, dass man sich dieser Problematik bereits bewusst ist. Ich denke es ist nur in ihrem Sinne diesen Mangel aus der Welt zu schaffen, da es ein ausschlaggebender Punkt sein kann ihre Hardware nicht zu kaufen oder gegen ein anderes Gerät zu ersetzen.

https://github.com/goecharger/go-eCharger-API-v2/issues/35

Wenn sie nichts dagegen haben, sammel ich gerne die Stimmen von ungluecklichen Nutzern im benannten Github Issue #35 um zu bekraeftigen, dass ihre Nutzerschaft nicht gluecklich mit diesem Umstand ist.

Mit freundlichen Grüßen

Sebastian M.

**Comment by 0xFEEDC0DE64** [comment]
als kurzfristigen work-around:
api key "wda", type boolean, "disable AccessPoint when cloud is connected for 30sec successfully"

**Comment by copacetic82** [comment]
> als kurzfristigen work-around: api key "wda", type boolean, "disable AccessPoint when cloud is connected for 30sec successfully"

oh super, vielen dank dafür!!! Freut mich sehr! :)

**Comment by syssi** [comment]
Keiner der beiden Befehle hat das WLAN-Netz aus der Luft verschwinden lassen:

```
mosquitto_pub -t '/go-eCharger/072246/wda' -m 'true' 
mosquitto_pub -t '/go-eCharger/072246/wda' -m 'false'
```

Zwischen dem anderen Verkehr (vom Charger) sieht es so aus:

```
/go-eCharger/072246/tma [29.75,34.125]
/go-eCharger/072246/fhz 49.833
/go-eCharger/072246/wda false
/go-eCharger/072246/fhz 50.008
...
/go-eCharger/072246/fhz 49.981
/go-eCharger/072246/wda true
/go-eCharger/072246/fhz 49.975
```

Mache ich etwas falsch? Versucht habe ich das Ganze mit der Firmware-Version 053.7 Beta.

**Comment by syssi** [comment]
```
curl -s "http://192.168.xx.xx/api/set?wda=true"
```

hat zum Erfolg geführt. Per

```
curl -s "http://192.168.xx.xx/api/status" | json_pp
```

lässt sich nachvollziehen, dass der Schalter per HTTP-API v2 betätigt werden kann, jedoch nicht per MQTT.

**Comment by gyroslav** [comment]
Wie verhält es sich mit dem API-Key "wda" genau?
Geht der AP von alleine wieder an, wenn die WB keine Verbindung zur Cloud hat?

**Comment by 0xFEEDC0DE64** [comment]
ja, genauergesagt, der AP geht nur dann aus, wenn die cloud verbindung schon 30sek lang besteht, und kommt sofort zurück, sobald die cloud verbindung unterbricht.

**Comment by gyroslav** [comment]
wo ist der Key dokumentiert? in der V2-API nicht.

**Comment by 0xFEEDC0DE64** [comment]
leider noch niergends, aus zeitgründen bin ich noch nicht dazu gekommen, setz einfach wda auf true oder false, wenn du willst, dass der AP mit erfolgreicher cloud verbindung abgedreht wird oder nicht.

**Comment by vistalba** [comment]
Bitte dringend beheben. 40MHz für einen Hotspot. Nicht deaktivierbar, wenn die Cloud nicht verwendet wird.

**Comment by phrozen77** [comment]
Ich lasse hier mal ein +1 da - hatte mich auch schon per Mail an den Support gewandt - der Hotspot ist auch ohne Cloud-Verbindung über sofern eine WLAN-Verbindung besteht...

Viele andere ESP-basierte Projekte machen den Hotspot auf wenn die WLAN-Verbindung nicht klappt oder auf Knopfdruck oder...

Warum der nur bei bestehender Cloud-Verbindung deaktiviert werden kann erschließt sich mir nicht so wirklich...

**Comment by roben** [comment]
Es wäre super, wenn sich hier noch eine Lösung zum dauerhaften Deaktivieren des Hotspots findet. Wir nutzen in der Firma 6 Go-E-Charger, die schlimmstenfalls alle einen eigenen Hotspot aufmachen, sobald deren Cloud mal nicht erreichbar ist. In einer Umgebung, die eh schon mit zu vielen WLAN-Signalen kämpft, ist das wirklich schwierig.
